### DISCUSSION TITLE
Support for updating conan.lock files

### DISCUSSION THREAD

**Discussion by japj** [renovatebot/renovate#21903] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Renovate supports updating conanfile.txt or conanfile.py.
However, there does not seem to be any support for updating the conan.lock file yet, which causes the lockfile to get out of sync.

Documentation on Conan lockfiles can be found at: https://docs.conan.io/2/tutorial/consuming_packages/intro_to_versioning.html#lockfiles

Is it possible to add support for updating conan.lock files?

**Comment by rarkins** [comment]
Converted to a feature request here: #21912
